### ISSUE TITLE
🎉 Community Update: X-AnyLabeling now supports PP-OCR model

### ISSUE THREAD

**Issue by CVHub520** [PaddlePaddle/PaddleOCR#16460]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

Hi PPOCR team and community! 👋

I wanted to share some exciting news with everyone. We've successfully integrated both PP-OCRv4 and PP-OCRv5 into [X-AnyLabeling](https://github.com/CVHub520/X-AnyLabeling), a comprehensive annotation tool that's designed to work seamlessly with the PPOCR ecosystem.

**What's X-AnyLabeling?**
X-AnyLabeling is a modern, user-friendly annotation software that provides full compatibility with PPOCR workflows. It supports:
- Text detection and recognition annotation [[Link]](https://github.com/CVHub520/X-AnyLabeling/blob/main/examples/optical_character_recognition/text_recognition/README.md)
- Key Information Extraction (KIE) tasks including SER (Semantic Entity Recognition) and RE (Relation Extraction) [[Link]](https://github.com/CVHub520/X-AnyLabeling/blob/main/examples/optical_character_recognition/key_information_extraction/README.md) )
- One-click export to PPOCR label formats for direct training [[Link]](https://github.com/CVHub520/X-AnyLabeling/blob/main/docs/en/user_guide.md#47-ppocr-format)
- Native support for PP-OCRv4 and PP-OCRv5 models [[Link]](https://github.com/CVHub520/X-AnyLabeling/blob/main/docs/en/model_zoo.md)

**Why am I sharing this?**
This isn't about reporting any issues (everything works great! 🚀), but rather a friendly heads-up to the community that there's now a dedicated annotation tool that's built specifically with PPOCR in mind. We hope this can help fellow researchers and developers who are working with PPOCR models streamline their annotation workflows.

The integration supports the complete PPOCR pipeline and makes it easier for users to prepare training data that's directly compatible with PPOCR training scripts. Looking ahead, I'm also considering integrating a one-click PPOCR training platform within X-AnyLabeling to create a complete end-to-end workflow from annotation to model training.

Thanks to the PPOCR team for creating such an amazing OCR framework that makes these integrations possible! 🙏

Feel free to check it out if you're looking for annotation tools that work well with your PPOCR workflows. 😄 

<img width="1920" height="1032" alt="Image" src="https://github.com/user-attachments/assets/f03c2ca3-ad47-4e54-9a70-cdf803d21921" />

### 🏃‍♂️ Environment (运行环境)

None

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

None

**Comment by cuicheng01** [comment]
@CVHub520 Hello! Thank you so much for sharing this exciting news with us. 🚀
We are thrilled to see that X-AnyLabeling has been successfully integrated with both PP-OCRv4 and PP-OCRv5. Your work will undoubtedly bring significant convenience to researchers and developers using PaddleOCR. 
We look forward to seeing how this tool helps streamline workflows and hope to see more collaboration opportunities in the future. Thank you again for your support and contribution to the PaddleOCR community!😄

**Comment by openvino-book** [comment]
🎉Big kudos for this fantastic collaboration across communities! 🤝 @CVHub520
